po/reuse-docs.pot

```diff
@@ -0,0 +1,1858 @@
+#: CHANGELOG.md
+#, markdown-text
+msgid "- The specification no longer mentions a bill of materials (SPDX Document)."
+msgstr ""
+
+#. type: Plain text
+#: README.md
+#, markdown-text, no-wrap
+msgid ""
+"<!--\n"
+"SPDX-FileCopyrightText: 2017 Free Software Foundation Europe e.V.\n"
+msgstr ""
+
+#. type: Title #
+#: README.md
+#, markdown-text, no-wrap
+msgid "REUSE documents"
+msgstr ""
+
+#. type: Plain text
+#: README.md
+#, markdown-text
+msgid "[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)  [![REUSE compliant](https://reuse.software/badge/reuse-compliant.svg)](https://reuse.software/)"
+msgstr ""
+
+#. type: Plain text
+#: README.md
+#, markdown-text
+msgid "We're working to make managing copyrights and licenses in free and open source software easier. This is a collection of tutorials, FAQs and specifications to demonstrate how to add copyright and license information to a project in ways which allow for more automation."
+msgstr ""
+
+#. type: Title ##
+#: README.md
+#, markdown-text, no-wrap
+msgid "Install"
+msgstr ""
+
+#. type: Plain text
+#: README.md
+#, markdown-text
+msgid "There's no installation here, only the plain text documents. The sister repository [reuse-website](https://github.com/fsfe/reuse-website) includes this repository as a submodule to display it on [reuse.software](https://reuse.software)."
+msgstr ""
+
+#. type: Title ##
+#: README.md
+#, markdown-text, no-wrap
+msgid "Usage"
+msgstr ""
+
+#. type: Plain text
+#: README.md
+#, markdown-text
+msgid "Go to [reuse.software](https://reuse.software) and enjoy :-) If your project follows the reuse guidelines, we encourage you to show that in your `README.md` and similar! Just register your project with the [REUSE API](https://api.reuse.software/) and copy the resulting badge code in your README file."
+msgstr ""
+
+#. type: Title ##
+#: README.md
+#, markdown-text, no-wrap
+msgid "Contribute"
+msgstr ""
+
+#. type: Plain text
+#: README.md
+#, markdown-text
+msgid "We'd love to get feedback on these practices, ideally in the form of pull requests which we can discuss around."
+msgstr ""
+
+#. type: Plain text
+#: README.md
+#, markdown-text
+msgid "We also accept and appreciate feedback by creating issues in the project or by sending an e-mail to the [public REUSE mailing list](https://lists.fsfe.org/mailman/listinfo/reuse)."
+msgstr ""
+
+#. type: Title ##
+#: README.md
+#, markdown-text, no-wrap
+msgid "License"
+msgstr ""
+
+#. type: Plain text
+#: README.md
+#, markdown-text
+msgid "The relevant documents in this repository are licensed under [Creative Commons Attribution-ShareAlike 4.0](https://creativecommons.org/licenses/by-sa/4.0)."
+msgstr ""
+
+#. type: YAML Front Matter: title
+#: faq.md
+#, no-wrap
+msgid "Frequently Asked Questions"
+msgstr ""
+
+#. type: Plain text
+#: faq.md
+#, markdown-text, no-wrap
+msgid "{{< toc >}}\n"
```

Comment:
Any way to exclude shortcodes to avoid them being translated?

Reply:
translators are used to seeing such things.  And like above, they can be marked in Weblate so that its clear to the translators.